src/foam/doc/DocBrowser.js

```diff
@@ -584,19 +584,7 @@ foam.CLASS({
     UNSELECTED_COLOR: '#FFFFCC'
   },
 
-  css: [
-    // TODO: remove '-' after ActionView when CSS naming fixed
-    foam.u2.CSS.create({
-      code: function() {
-        /*
-               ^ { width:1200px; margin: 20px; }
-               ^ canvas { border: 1px solid black; }
-               ^ .foam-u2-ActionView- { margin: 10px; }
-               ^ input[type='range'] { width: 400px; }
-               */
-      }
-    })
-  ],
+  css: ^ { width:1200px; margin: 20px; } ^ canvas { border: 1px solid black; } ^ .foam-u2-ActionView- { margin: 10px; } ^ input[type='range'] { width: 400px; },
```

Comment:
This needs to be quoted, preferably in a multi-line quote (`). Doesn't this cause a syntax error?

Reply:
No, it does not.